serialization.md

```diff
@@ -468,62 +462,89 @@ Merkle Patricia Tree
                         | MerkleExtension ( ByteArray, MerkleTree )
                         | MerkleLeaf      ( ByteArray, String )
                         | ".MerkleTree"
-                        | ".MerkleBranch"            [function]
  // -----------------------------------------------------------
```

Comment:
Put `.MerkleTree` as the first production here, and adjust the `-----` to extend the length of the new last production `MerkleLeaf ( ByteArray, String)`